deeprank/learn/DataSet.py

```diff
@@ -1168,6 +1168,7 @@ def load_one_molecule(self, fname, mol=None):
         try:
             target = mol_data.get('targets/' + self.select_target)[()]
         except Exception:
+            target = None
             logger.exception(f'No target value for: {fname} - not required for the test set')
```

Comment:
Make more sense to change logger.exception to logger.warning?

Reply:
Thanks Liang, I agree.